testimony.yaml

```diff
@@ -67,7 +67,7 @@ CaseComponent:
     - katello-agent
     - katello-tracer
     - LDAP
-    - Leappintegration
+    - LeappIntegration
```

Comment:
We need to rename this back, so either need to cherrypick this when https://github.com/SatelliteQE/robottelo/pull/12639 merges, or manually change it here

Reply:
done

Reply:
@Gauravtalreja1 @jyejare @devendra104 
One check is failing due to component name issue

Reply:
sorry my bad, I haven't changed component name from `testimony.yaml`